src/main/java/cz/fi/muni/pa165/Model/Entity/ApplicationApprovedRecord.java

```diff
@@ -59,4 +60,30 @@ public Date getTo() {
     public RentApplication getApplication() {
         return application;
     } 
+    
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if ((o == null) || !(o instanceof ApplicationApprovedRecord)) {
+            return false;
+        }
+        ApplicationApprovedRecord other = (ApplicationApprovedRecord) o;
+        if (getFrom() != null ? !getFrom().equals(other.getFrom()) : other.getFrom() != null) {
+            return false;
+        }
+        if (getTo() != null ? !getTo().equals(other.getTo()) : other.getTo() != null) {
+            return false;
+        }
+        if (application != null ? !application.equals(other.getApplication()) : other.getApplication() != null) {
+            return false;
+        }
+	return true;
+    }
+
+    @Override
+    public int hashCode() {
+        return new HashCodeBuilder().append(getFrom()).append(getTo()).append(getApplication().getId()).toHashCode();
```

Comment:
@jakubsarmir , what is your opinion on this? Inconsistency could always be a choice (even though I really don't like that one).